client/webserver/site/src/js/markets.js

```diff
@@ -198,6 +199,35 @@ export default class MarketsPage extends BasePage {
     bind(page.marketSearch, 'change', () => { this.filterMarkets() })
     bind(page.marketSearch, 'keyup', () => { this.filterMarkets() })
 
+    // Load the user's layout preferences.
+    const setChartRatio = r => {
+      if (r > 0.7) r = 0.7
+      else if (r < 0.25) r = 0.25
+      page.marketChart.style.height = `${r * 100}%`
+      this.chart.resize()
+    }
+    const chartDivRatio = State.fetch(chartRatioKey)
+    if (chartDivRatio) {
+      setChartRatio(chartDivRatio)
+    }
+    // Bind chart resizing.
+    bind(page.chartResizer, 'mousedown', e => {
+      if (e.button !== 0) return
+      e.preventDefault()
+      const trackMouse = ee => {
+        ee.preventDefault()
+        const box = page.rightSide.getBoundingClientRect()
+        const h = box.bottom - box.top
+        var chartRatio = (ee.pageY - box.top) / h
+        setChartRatio(chartRatio)
+        State.store(chartRatioKey, chartRatio)
```

Comment:
Seems like it should store the key on `mouseup` rather than on each mouse movement.  Practically maybe it doesn't matter?  Oh, I guess the `mousemove` event fires when the mouse **stops** after movement -- that works then if that is correct.